test/controllers/contactDetails/CharityContactDetailsControllerSpec.scala

```diff
@@ -59,8 +59,8 @@ class CharityContactDetailsControllerSpec extends SpecBase with BeforeAndAfterEa
   private val controller: CharityContactDetailsController = inject[CharityContactDetailsController]
 
   private val requestArgs = Seq(
-    "mainPhoneNumber"        -> "07700 900 982",
-    "alternativePhoneNumber" -> "07700 900 982",
+    "mainPhoneNumber"        -> s"$daytimePhone",
```

Comment:
No need for s", as above. Anywhere else you have this pattern just replace with val